src/relative-time-element.ts

```diff
@@ -382,6 +383,10 @@ export class RelativeTimeElement extends HTMLElement implements Intl.DateTimeFor
     this.setAttribute('format-style', value)
   }
 
+  get noTitle(): boolean {
+    return this.hasAttribute('no-title') && this.getAttribute('no-title') === 'true'
+  }
```

Comment:
Non-blocking:
- Is there any reason to make this private?
- Is there any reason to add a setter (to support e.g. `relativeTime.noTitle = false`)?

(I can’t think of any good reasons.)